ERCS/erc-xxxx.md

```diff
@@ -0,0 +1,243 @@
+| sender                        | DATA, 20 Bytes | Address of the Smart Contract Account making the transaction   |
+| deployer                      | DATA, 20 Bytes | Address of the Deployer - account factory contract             |
+| deployerData                  | DATA           | Data that is provided to the Deployer contract                 |
+| paymaster                     | DATA, 20 Bytes | Address of the Paymaster contract                              |
+| paymasterData                 | DATA           | Data that is provided to the Paymaster contract                |
+| executionData                 | DATA           | Data that is provided to the Account contract for execution    |
+| nonce                         | QUANTITY       | A 256 bit nonce. Use of `nonce > 2^64` is defined in RIP-7712  |
+| builderFee                    | QUANTITY       | Value passed from sender or paymaster to the `coinbase`        |
+| maxPriorityFeePerGas          | QUANTITY       | The maximum gas price to be included as a tip to the validator |
+| maxFeePerGas                  | QUANTITY       | The maximum fee per unit of gas                                |
+| validationGasLimit            | QUANTITY       | Gas provided for the transaction account validation frame      |
+| paymasterValidationGasLimit   | QUANTITY       | Gas provided for the transaction paymaster validation frame    |
+| paymasterPostOpGasLimit       | QUANTITY       | Gas provided for the transaction paymaster `postOp` frame      |
+| callGasLimit                  | QUANTITY       | Gas provided for the transaction execution frame               |
+| accessList                    | OBJECT         | An EIP-2930 compatible Access List structure                   |
+| EIP-7702 authorizations (WIP) | ARRAY          | An EIP-7702 compatible list of contracts injected into EOAs    |
+| authorizationData             | DATA           | Data that will be used by the Account to verify transaction    |
+
+### Add RIP-7560 support for `eth_getTransactionReceipt`
+
+For an RIP-7560 transaction included in a block, return also the values specific to this transaction type
+in addition to all the existing fields.
+
+Parameters:
+
+1. DATA, 32 Bytes - hash of a transaction
+
+Returns:
+
+1. OBJECT - A transaction receipt object, or `null` when no receipt was found:
+
+Fields specific to an RIP-7560 transaction receipt:
+
+| Name                       | Type           | Description                                                                    |
+|----------------------------|----------------|--------------------------------------------------------------------------------|
+| sender                     | DATA, 20 Bytes | Address of the sender of this transaction                                      |
+| paymaster                  | DATA, 20 Bytes | Address of the Paymaster if it is paying for the transaction, `null` otherwise |
```

Comment:
put all paymaster fields together: if paymaster exists, then all other paymaster fields MUST exist.
if paymaster is missing, then all others MUST NOT exist